### PR TITLE
allow audio uploads

### PR DESCRIPTION
**Pull request by ruslandoga**

### CODE REVIEW

File: lib/t/voicemail.ex [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def audio_s3_url do
+    Media.user_s3_url()
+  end
+
+  def audio_upload_form(content_type) do
+    Media.sign_form_upload(
+      key: Ecto.UUID.generate(),
+      content_type: content_type,
+      # 50 MB
+      max_file_size: 50_000_000,
```
Comment (ruslandoga) [inline lib/t/voicemail.ex:14]: or we can make the client provide us the size it want to send

File: lib/t_web/controllers/media_controller.ex [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
+  # if it's an image -> it's for profile story photo
+  defp upload_form("image/" <> _rest = content_type) do
     {:ok, %{"key" => key} = fields} = Accounts.photo_upload_form(content_type)
-    url = Accounts.photo_s3_url()
+    %{url: Accounts.photo_s3_url(), key: key, fields: fields}
+  end
 
-    json(conn, %{url: url, key: key, fields: fields})
+  # if it's an audio/aac -> it's for voicemail
+  defp upload_form("audio/aac" = content_type) do
```
Comment (akhrail1996) [inline lib/t_web/controllers/media_controller.ex:22]: any rationale for choosing `aac` specifically?

**Review by akhrail1996 — APPROVED** [review]
thanks 🙏

File: lib/t_web/controllers/media_controller.ex (hunk 2 above)
Comment (ruslandoga) [reply lib/t_web/controllers/media_controller.ex:22]: does apple support anything else?

File: lib/t/voicemail.ex (hunk 1 above)
Comment (akhrail1996) [reply lib/t/voicemail.ex:14]: I will experiment with recordings on iOS to see the file sizes, but this should probably suffice for now

File: lib/t_web/controllers/media_controller.ex (hunk 2 above)
Comment (akhrail1996) [reply lib/t_web/controllers/media_controller.ex:22]: https://stackoverflow.com/questions/6867994/what-are-the-formats-supported-in-avaudiorecorder-for-recording-sound
seems to support some, yeah

Comment (ruslandoga) [reply lib/t_web/controllers/media_controller.ex:22]: for recording ...

**Pull request merged**